.github/workflows/build.yml

```diff
@@ -13,10 +13,25 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install deps
         run: |
-          sudo apt-get install g++ gcc libxxf86vm-dev libgl1-mesa-dev libbz2-dev libpng-dev libjpeg-dev zlib1g-dev -qyy
+          sudo apt-get install -qyy cmake g++ gcc libbz2-dev libpng-dev libjpeg-dev wget zlib1g-dev \
+            libasound2-dev libdbus-1-dev libegl1-mesa-dev libgl1-mesa-dev libgles2-mesa-dev libglu1-mesa-dev libibus-1.0-dev libpulse-dev libsndio-dev libudev-dev libwayland-dev libx11-dev libxcursor-dev libxext-dev libxi-dev libxinerama-dev libxkbcommon-dev libxrandr-dev libxss-dev libxt-dev libxv-dev libxxf86vm-dev 
+
+      - name: Build SDL
+        run: |
+          wget https://www.libsdl.org/release/SDL2-2.0.14.tar.gz
```

Comment:
Uh, why not just install it from the repos?

Reply:
Because SDL 2.0.14 is required (dunno why, that might be relaxed later) while Ubuntu 18.04 has only 2.0.12